SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestPossession/TestPossessionPlayerController.cpp

```diff
@@ -60,13 +60,38 @@ void ATestPossessionPlayerController::RemotePossessOnClient_Implementation(APawn
 
 void ATestPossessionPlayerController::RemotePossessOnServer(APawn* InPawn)
 {
+	check(HasAuthority());
+
 	URemotePossessionComponent* Component =
 		NewObject<URemotePossessionComponent>(this, URemotePossessionComponent::StaticClass(), TEXT("CrossServer Possession"));
 	Component->Target = InPawn;
 	Component->RegisterComponent();
 	BeforePossessionWorkerId = GetCurrentWorkerId();
 }
 
+void ATestPossessionPlayerController::RemovePossessionComponent()
+{
+	URemotePossessionComponent* Component =
+		Cast<URemotePossessionComponent>(GetComponentByClass(URemotePossessionComponent::StaticClass()));
+	if (Component != nullptr)
+	{
+		Component->DestroyComponent();
+	}
+}
+
+void ATestPossessionPlayerController::UnlockAllTokens()
+{
+	USpatialNetDriver* NetDriver = Cast<USpatialNetDriver>(GetNetDriver());
+	if (NetDriver != nullptr && NetDriver->LockingPolicy)
```

Comment:
`NetDriver != nullptr` is this ever legit? I don't think we can run this if we are not running on spatial (which is the only valid reason for this to be null I can think of in this context). Could be a `check`